src/index.html

```diff
@@ -22,13 +23,13 @@
           <nav>
             <ul class="main-nav">
               <li>
-                <a href="" class="logo"><img src="" alt="Logo"/></a>
+                <a href="index.html" class="logo"><img src="" alt="Logo"/></a>
               </li>
               <li><a href="">link</a></li>
               <li><a href="">link</a></li>
               <li><a href="">link</a></li>
               <li><a href="">link</a></li>
-              <li><a href="" class="cta">link</a></li>
+              <li><a href="login.html" class="cta">Login</a></li>
```

Comment:
For the mobile screen, login anchor tag overflows the layout. Therefore you have a horizontal scroll bar. It is probably still a work in process. 🙂